packages/components/src/side_scroller/index.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "nebenan-ui-kit";
+
+.root {
+  overflow: hidden;
+  position: relative;
+}
+
+.container {
+  margin-bottom: -30px;
```

Comment:
i personally don't like negative margin usage, isn't there any way to do it without a negative margin?

Reply:
I agree. My approach would be to get rid of margin & padding and set overflow-x to hidden.